app/Listeners/LoginListener.php

```diff
@@ -14,7 +17,6 @@ class LoginListener
      */
     public function __construct()
     {
-        //
```

Comment:
An empty function might break tests, other things. Let's re-add the comment slashes.